pkgs/development/python-modules/ffc/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchurl
+, python3Packages
+, dolfin
+}:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  pname = "ffc";
+  inherit (dolfin) version;
+
+  src = fetchurl {
+    url = "https://bitbucket.org/fenics-project/ffc/downloads/ffc-${version}.tar.gz";
+    sha256 = "1zdg6pziss4va74pd7jjl8sc3ya2gmhpypccmyd8p7c66ji23y2g";
+  };
+
+  nativeBuildInputs = [
+    pybind11
+  ];
+
+  propagatedBuildInputs = [
+    dijitso
+    fiat
+    numpy
+    six
+    sympy
+    ufl
+    setuptools
+  ];
+
+  checkInputs = [ pytest ];
+
+  preCheck = ''
+    export HOME=$PWD
+    rm test/unit/ufc/finite_element/test_evaluate.py
```

Comment:
please add a comment saying way this is necessary

Reply:
I have no idea why that is necessary. It was in the original code, and I kept it.